tree/ntuple/v7/src/RField.cxx

```diff
@@ -892,7 +892,7 @@ ROOT::Experimental::RFieldBase::Create(const std::string &fieldName, const std::
    } catch (RException &e) {
       auto error = e.GetError();
       if (createContext.GetContinueOnError()) {
-         std::unique_ptr<RFieldBase>(std::make_unique<RInvalidField>(fieldName, canonicalType, error.GetReport()));
+         return { std::make_unique<RInvalidField>(fieldName, canonicalType, error.GetReport()) };
```

Comment:
I agree with `clang-format` that this looks odd... At least the spaces should be removed, not sure if there is value in spelling out the conversion to `std::unique_ptr<RFieldBase>` that was there before...

Reply:
To me it makes little sense to spell out the `unique_ptr` conversion; if anything we should spell out the conversion to `RResult`.. (as for the spacing, I find it ugly to not have it but we have a clang-format to avoid aesthetic preferences conflicts, so I'll abide by it :P)

Reply:
That said, since spelling out RResult is much more verbose I think I'll just keep it as it was and call it a day